src/dbnode/runtime/runtime_options.go

```diff
@@ -67,6 +68,7 @@ type options struct {
 	writeNewSeriesAsync                  bool
 	writeNewSeriesBackoffDuration        time.Duration
 	writeNewSeriesLimitPerShardPerSecond int
+	encodersPerBlockLimit                *atomic.Uint32
```

Comment:
all options we pass around like are modeled as immutable. Lets continue that pattern please.